sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/implementation/HealthcareEntityPropertiesHelper.java

```diff
@@ -44,7 +46,7 @@ public static void setText(HealthcareEntity healthcareEntity, String text) {
         accessor.setText(healthcareEntity, text);
     }
 
-    public static void setCategory(HealthcareEntity healthcareEntity, EntityCategory category) {
+    public static void setCategory(HealthcareEntity healthcareEntity, String category) {
```

Comment:
why did we drop the enum?

Reply:
Healthcare entity category is not same as the Named(Categorized) Entity category. Agreed with .NET approach.